### PR TITLE
Add changelog for elastic-opentelemetry-instrumentation-openai 0.1.0

### PR DESCRIPTION
**Pull request by xrmx**

### CODE REVIEW

File: instrumentation/elastic-opentelemetry-instrumentation-openai/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Elastic OpenTelemetry Instrumentation OpenAI
+
+## v0.1.0
+
+Initial release with the following features:
+- `sync` and `async` chat completions
+- Streaming support
+- Functions calling with tools
+- Client side metrics
+- Following 1.27.0 Gen AI Semantic Convention
```
Comment (estolfo) [inline instrumentation/elastic-opentelemetry-instrumentation-openai/CHANGELOG.md:10]: Should we put a reference here?

File: instrumentation/elastic-opentelemetry-instrumentation-openai/README.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ messages = [
 chat_completion = client.chat.completions.create(model="gpt-4o-mini", messages=messages)
 ```
 
-### Instrumentation specific configuration variables
+### Instrumentation specific configuration environment variables
```
Comment (estolfo) [inline instrumentation/elastic-opentelemetry-instrumentation-openai/README.md:49]:
```suggestion
### Instrumentation specific environment variable configuration
```

**Review by estolfo — APPROVED** [review]

File: instrumentation/elastic-opentelemetry-instrumentation-openai/CHANGELOG.md (hunk 1 above)
Comment (estolfo) [reply instrumentation/elastic-opentelemetry-instrumentation-openai/CHANGELOG.md:10]: Also, Conventions -> Convention**s**

Comment (xrmx) [inline instrumentation/elastic-opentelemetry-instrumentation-openai/CHANGELOG.md:10]:
```suggestion
- Following 1.27.0 [Gen AI Semantic Conventions](https://opentelemetry.io/docs/specs/semconv/gen-ai/)
```

Comment (xrmx) [reply instrumentation/elastic-opentelemetry-instrumentation-openai/CHANGELOG.md]: Added, thanks!

**Pull request merged**